lecture-notes/2-vcs.md

```diff
@@ -1,11 +1,12 @@
 ---
+layout: spec
 ---
 
 # Lecture Notes: Version Control Systems
 
-## Lecture 1
+# Lecture 1
```

Comment:
It looks a bit jarring to have these headings be the same size as the page title above. I assume you changed this to make the sidebar look better, but would it be possible to have separate styling for the page title now that these are H1s?

Reply:
I understand the concern — my reasoning is that only the top of the page is affected, and students will spend very little time looking at that.

If this is a _blocking_ concern, then here are two options we can consider:
1. Keep the changed header levels but add a custom style to make the headings look different. I don't prefer this option for the same reason that I pushed back on CSS customizations in https://github.com/tekknolagi/isdt/pull/22#issuecomment-1033960702. We *can* add a page-specific style just to make the title of the page look bigger than a typical `<h1>`, or we would need to add some global CSS customization.
2. Revert the header levels, and update Primer Spec to show the left-border to group top-level H2 items. I'll need some time to figure out how to achieve it in an aesthetic manner.